descriptors/ModuleDescriptor-template.json

```diff
@@ -59,6 +59,7 @@
           "pathPattern": "/records-editor/records/{id}",
           "permissionsRequired": ["records-editor.records.item.delete"],
           "modulePermissions": [
+            "inventory-storage.authorities.item.delete",
```

Comment:
I'd prefer to have this module permission in mod-srm for the endpoint with `change-manager.records.post` permission. This is better just because mod-qm doen't have dependencies on mod-inventory-storage

Reply:
True genius. Opened pull request for `srm` module
https://github.com/folio-org/mod-source-record-manager/pull/634